sunflower-land/sunflower-land
# [FEAT] Kingdom NPCs

# Description

New NPCs that unlock in later stages of the game (late Petal Paradise)

- Victoria (Coin deliveries)
- Jester (Ticket deliveries - Only live on mainnet 14th June)
- Gambit (SFL deliveries)

This PR introduces the new Kingdom NPCs and setups deliveries for them. Including:

- Custom Dialogue Text
- Custom Gifts (Victoria + Jester) & Wearable Rewards

<img width="456" alt="Screenshot 2024-06-03 at 5 26 08 PM" src="https://github.com/sunflower-land/sunflower-land/assets/11745561/0a555d8f-20c9-4611-93f7-7afe4d9e1af4">

<img width="270" alt="Screenshot 2024-06-03 at 5 56 05 PM" src="https://github.com/sunflower-land/sunflower-land/assets/11745561/40fb98c5-f294-41f4-90c0-98b4d3ea9a51">
<img width="292" alt="Screenshot 2024-06-03 at 5 56 23 PM" src="https://github.com/sunflower-land/sunflower-land/assets/11745561/414c5220-2732-48c7-8123-fa26ff1c47b5">
<img width="216" alt="Screenshot 2024-06-03 at 5 56 21 PM" src="https://github.com/sunflower-land/sunflower-land/assets/11745561/22232d1a-39e2-484c-92b1-3409ecd5fc49">

## How to test?

1. Run the API
2. Make sure you have unlocked enough expansions (e.g. made it to Desert island)
3. Go to the Kingdom
4. Complete Deliveries
5. Gift
